### DISCUSSION TITLE
Option to disable coloring in stat panel

### DISCUSSION THREAD

**Discussion by hasso** [grafana/grafana#38154] [Dashboards, Panels & Visualizations]
In stat panel you have to choose what to color – a value or background, old singlestat panel had an option to color neither. I have a lot of stat panels in which coloring just doesn't make sense, these are just informal and the only option I have at the moment is to tinker with custom color for baseline. This kind of tricks shouldn't be necessary for such simple task.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
